### PR TITLE
✨[Amp Story] [Page attachments] Remove background gradient

### LINKED ISSUE #33767: [Page attachments] Remove background linear gradient
There's a slight linear gradient that is removed in the new CTA design.

Screen Shot 2021-04-12 at 1 33 11 PM

@ampproject/wg-stories

### PR DESCRIPTION
**Pull request by processprocess**
Context / Fixes #33767

[demo](https://philipbell-stamp-ui.web.app/examples/amp-story/attachment.html)

### CODE REVIEW

File: extensions/amp-story/1.0/amp-story-open-page-attachment.css
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-open-page-attachment.css:249]: Do you need the [active]? If it's for CSS specificity, I think we could do `.i-amphtml-story-page-open-attachment.i-amphtml-story-outlink-page-attachment` so the background never renders at all, even on inactive pages

Comment (processprocess) [reply extensions/amp-story/1.0/amp-story-open-page-attachment.css]: Revised this to remove the background on all CTA variations in the experiment.

File: extensions/amp-story/1.0/amp-story-open-page-attachment.js
Comment (gmajoulet) [inline extensions/amp-story/1.0/amp-story-open-page-attachment.js:59]: we still need to add the prefix i-amphtml to ensure publishers can't target it

**Review by gmajoulet — COMMENTED** [review]
Shouldn't we keep it for the inline attachments, both regular and image UI?

File: extensions/amp-story/1.0/amp-story-open-page-attachment.js
Comment (processprocess) [reply extensions/amp-story/1.0/amp-story-open-page-attachment.js]: Added it in all other areas that class is used too.

**Review by gmajoulet — COMMENTED** [review]
The implementation LGTM but:
> Shouldn't we keep it for the inline attachments, both regular and image UI?

**Comment by processprocess** [comment]
> Shouldn't we keep it for the inline attachments, both regular and image UI?

Met with @hongcatlover and we're removing it on inline and outlink CTAs for now.
It needs background protection since we can't 100% rely on educating users. 
We're ideating on how we want to do that and I'll follow up with it once the design is approved.

**Review by gmajoulet — APPROVED** [review]
If Hong approved it's good with me, but if any publisher relied on it for their templates, we might make a11y a bit weaker when rolling out this new design.

**Comment by processprocess** [comment]
> we might make a11y a bit weaker when rolling out this new design.

Hong and I have designs for background protection this in the works :)

**Pull request merged**
